sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/CBDE/CbdeHandler.cs

```diff
@@ -23,23 +23,32 @@
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using SonarAnalyzer.Helpers;
 using System.IO;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Text;
 using Microsoft.CodeAnalysis.Diagnostics;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Text;
 using System.Threading;
+using System.Xml.Linq;
+using System.Xml;
 
 namespace SonarAnalyzer.CBDE
 {
     public class CbdeHandler
     {
+        public enum MockType
```

Comment:
I believe we don't need this
- we can just have a constructor w/o the CBDE binary path, and then the `Initialize` does the magic- just like until now
- and we can have a constructor that gets an already created binaryPath which is passed from the tests

I would avoid any sort of complex mocking logic inside the production code